SimpleColorApp/SimpleColorApp/ContentView.swift

```diff
@@ -14,7 +14,7 @@ struct ContentView: View {
   @State private var blueSliderValue: Double
   @State private var boxColor:Color
   
-  init(red: Double = 0.6, green: Double = 0.3, blue: Double = 0.8) {
+  init(red: Double = 0.7, green: Double = 0.7, blue: Double = 0.7) {
```

Comment:
By setting a default value here you're making the parameters optional...so
````
init(red: 03)
````
is valid and would result in red = 0.3, green = 0.7 and blue = 0.7
The compiler is smart enough to fill in the others (but it will yell at you if you try to switch up the order)